stdlib/public/stubs/Stubs.cpp

```diff
@@ -378,6 +378,28 @@ T _swift_strto(const char *nptr, char **endptr) {
   return ParsedValue;
 }
 
+#if defined(__OpenBSD__) || defined(_WIN32) || defined(__CYGWIN__) || defined(__HAIKU__)
+#define need_swift_strtod_l
```

Comment:
I moved this block before the functions, and added these `need_swift_strto*` "variables". Each function is then gated by the definition of the appropriate variable.

Reply:
The upfront configuration makes sense.  The only thing I find a bit difficult to read is the difference between the `need_*` and `strto*` renaming.  I don't really have much of a good solution to that other than perhaps use upper case macro names for the `NEED_*`?

Reply:
Uppercase works for me. I wondered about that when doing it initially.

Reply:
updated